src/wattsi.pas

```diff
@@ -1972,10 +1975,12 @@ procedure PreProcessCanIUseData(const CanIUseJSONFilename: AnsiString);
    States: set of TImplState;
    NewState: TImplGoodState;
 begin
-   Writeln('Parsing caniuse.com data...');
+   if (not(Quiet)) then
+      Writeln('Parsing caniuse.com data...');
```

Comment:
I don't know this language, but isn't it possible to define a Log helper that does nothing if not quiet?